subsys/bluetooth/host/gatt.c

```diff
@@ -5696,3 +5697,153 @@ void bt_gatt_disconnected(struct bt_conn *conn)
 	remove_cf_cfg(conn);
 #endif
 }
+
+struct svc_uuid_data_state {
+	struct bt_data *data;
+	uint8_t *data_buf;
+	const size_t data_buf_size;
+	bool truncated;
+};
+
+/* This is an array of UUID values that we return via the
+ * bt_gatt_get_svc_uuid_data for advertisements
+ */
+static uint16_t svc_uuid16_data_skip_list[] = {
```

Comment:
I suggest "exclude list" instead of "skip list". "skip list" can be confused with the type of data structure with that name.